### PR TITLE
Add http filter to versioned docs

### PR DESCRIPTION
**Pull request by jsvd**

### CODE REVIEW

File: docs/versioned-plugins/filters/http-v0.1.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+===== `target`
+
+  * Value type is {logstash-ref}/configuration-file-structure.html#hash[hash]
+  * There is no default value
+
+Define the target field for placing the body of the HTTP response. If this setting is omitted, the data will be stored in the "body" field.
+
+[id="{version}-plugins-{type}s-{plugin}-target_headers"]
+===== `target`
+
```
Comment (karenzone) [inline docs/versioned-plugins/filters/http-v0.1.0.asciidoc]: I see two headings called `target` (lines 104 and 112)

File: docs/versioned-plugins/filters/http-v0.1.0.asciidoc [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+This plugin supports the following configuration options plus the <<{version}-plugins-{type}s-{plugin}-common-options>> described later.
+
+[cols="<,<,<",options="header",]
+|=======================================================================
+|Setting |Input type|Required
+| <<{version}-plugins-{type}s-{plugin}-body>> | String, Array or Hash |No
+| <<{version}-plugins-{type}s-{plugin}-body_format>> |{logstash-ref}/configuration-file-structure.html#string[string]|No
+| <<{version}-plugins-{type}s-{plugin}-headers>> |{logstash-ref}/configuration-file-structure.html#hash[hash]|No
+| <<{version}-plugins-{type}s-{plugin}-query>> |{logstash-ref}/configuration-file-structure.html#hash[hash]|No
```
Comment (karenzone) [inline docs/versioned-plugins/filters/http-v0.1.0.asciidoc]: `query` is referenced, but not defined.

Comment (karenzone) [reply docs/versioned-plugins/filters/http-v0.1.0.asciidoc]: @jsvd Hmmm...  I guess technically, these comments belong in the filters-http repo.

Comment (jsvd) [reply docs/versioned-plugins/filters/http-v0.1.0.asciidoc]: Agreed, I've added these to https://github.com/logstash-plugins/logstash-filter-http/pull/7

**Pull request merged**